programs/client/Client.cpp

```diff
@@ -2694,6 +2698,10 @@ class Client : public Poco::Util::Application
 
 int mainEntryClickHouseClient(int argc, char ** argv)
 {
+#if WITH_COVERAGE
+    ::detail::Writer::instance().onClientInitialized();
```

Comment:
Remove Writer from detail namespace, detail namespaces should not be used by clients. Consider just Writer::instance().

Reply:
This code is not intended to be used by clients (only by coverage runtime developer), so internal namespace is OK as it emphasises runtime's task.